actor/kvcounter-wasmcon2023/src/lib.rs

```diff
@@ -237,6 +237,7 @@ fn write_http_response(
 
 /// The response that is sent by the API after an operation
 #[derive(Debug, Serialize)]
+#[serde(untagged)]
```

Comment:
For the curious, this is here because otherwise it serialized as a tagged enum: `{"Success":{"counter":5}}` when the UI was just expecting `{"counter":5}`